### DISCUSSION TITLE
Help... Frustrating Experience with GitHub Student Developer Pack Verification Process

### DISCUSSION THREAD

**Discussion by Yan-Yu-Lin** [community/community#51568] [GitHub Education]
Hi GitHub Education Community,

I recently applied for the GitHub Student Developer Pack and provided several documents, including my student ID and multiple screenshots as proof of my current enrollment in Electronics at NTUT. Unfortunately, my application was denied, and I'm feeling quite upset about the entire experience.

The verification process has been time-consuming and frustrating, and I encountered difficulties in providing complete documentation. Despite my efforts, the GitHub Education Team accused me of attempting to manipulate the documents and images, which I find deeply disheartening and offensive.

I have tried reaching out to the GitHub Education Team directly but was unable to get a response, as they directed me to this community for assistance. I understand the need for a strict verification process to ensure that only eligible students receive the benefits of the Pack. However, I believe there should be a more accessible and user-friendly process in place, especially for students who face challenges in providing extensive documentation.

I kindly request the GitHub Education Community to provide guidance on how to resolve this issue and successfully verify my academic affiliation. Additionally, I hope that my experience can serve as feedback to improve the verification process for future applicants and prevent others from being unfairly accused of manipulation.

Thank you for your support!

**Comment by samus-aran** [comment]
Hi @Yan-Yu-Lin !

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

If you need help with applying please check out our [Apply to GitHub Global Campus for Students](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student) as well as our [Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved). We aim to assist our students by providing rejection reasons throughout the application process so that you can take steps to rectify and re-apply. But we also understand that there are times where you need a little more guidance and we're here to nudge you in the right direction. 

What were the most recent rejection reasons and we can go from there. ⭐

**Reply by AliAmmarNaseem** [reply]
I have been trying to apply for the student developer pack for many days now, but unfortunately, I have been rejected 20 times after uploading an image. I have tried uploading different documents like my student ID card, LMS screenshot, fee challan and semester schedule but still not getting approval.

I am attaching a real image issued by my institute which is not blurry and still facing problems with approval. When I upload from gallery, they say :
"The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:
Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy."

"When I use device camera and capture image then get following error:
Unfortunately, we weren't able to approve your educational discount request for {my username}

What happened?
Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved."

What should I do now? Please help me.

Thank you for your time and consideration.

**Comment by ghost** [comment]
I thought I could share my experience with applying for student status. I just have been rejected a third time. It looks to me like GitHub does it on purpose to force people to pay. I can't remember why I was rejected the first time but probably something stupid like the second and the third time. The second time I sent screenshots confirming my enrolment, they rejected it telling me the screenshots had been edited, and asked me to send pictures next time. The third time I took pictures and the same happened! The image was straight from the phone and I received an answer from them literally 10 minutes later telling me that the picture had been modified. This is what I'm going to do. I will delete my account from here and never use this crap again. I'll go somewhere else and pay if I have to but you're getting nothing from me.  This is what you get when you treat people like this.

**Comment by MBanatovski** [comment]
The system is really frustrating indeed, it asks for a camera on my computer, mine is super blurry 2010 logitech cam so doesn't comply. I try doing on phone click back facing camera and it still show me the front camera. Also super picky, "school logo is not sufficient", Like I have an email with the school domain wtf else do you want?????/

**Reply by MBanatovski** [reply]
Most likely so people pay. So basically another company going for greed while trying to seem "nice".

**Comment by juanpflores** [comment]
Closing this discussion as I can see that you've got access to the benefits already ❇️

**Comment by AleksaZCodes** [comment]
Same here, I applied for the third time and got rejected, not as much as you all have, but I see where the strict rules may make it hard for me to get approved.

First of all, the first document I provided was a certificate of school attendance or so, the document my school provides for other benefits. Apparently, the issue with it was it being written in Cyrillic, because I'm from Serbia, makes sense, and they said if I have documents on a local language, I should provide the English transcript. Note: my profile and bill information names are in Latin, as that's more frequently used online).

On the second request I took a different approach. Instead of that certificate, I scanned (with my phone) my European Youth Card alongside my student id card ("đačka knjižica" in Serbian) and transcribed it using Google Lens before stitching them all in Photoshop and taking the picture using a virtual camera. It's physically impossible to take a non-blurry picture of all the documents, not to mention the transcript which I don't have physically. It resulted in a location rejection because I connected to a network where it gave me a misleading location.

On the third attempt, after changing the layout for the automatic systems to be happy, it rejected me without details. I'm aware that scanned documents may be problematic, but at the same time when I take a blurry image that has my name written like "Алекса Здравковић" instead of "Aleksa Zdravković" in Serbian, a whole new reason to reject me will appear.

Deeply disappointed, the only thing I'm left with is to ask the GitHub Education Community to help me with my application and to potentially obtain my Github Student Developer Pack.

Thanks in advance!

**Comment by teressacodes** [comment]
Why must it be a screenshot? You should be able to upload a document. Very disppointed with this process and I would like someone to contact me about this asap.

**Comment by Yc-Han** [comment]
Cannot agree more. The process is extremely annoying. I once uploaded my student certificate, and it was rejected because I did not update my billing information or etc. And it keeps saying I cannot upload the same photo for verification again. Isn't that ridiculous?
Also, not being able to upload documents is a total SHAME for GitHub. I can't believe this is the process that programmers would EVER develop!
I mainly need the student benefits for my Copilot usage. They try to have us pay the 10$/month with the most bureaucratic process ever appeared on this platform obviously.

**Comment by Shifath472533** [comment]
Very much annoying process.

**Comment by LeeYi-master** [comment]
Very much annoying process indeed.
